### ISSUE TITLE
Unstable cpu usage and framerate drop

### ISSUE THREAD

**Issue by Avavafgfgh** [jeffshee/gnome-ext-hanabi#96]
**Description**
Unstable cpu usage and framerate drop

**OS and Shell Information**
Please provide your OS name and version, GNOME Shell version, and the session type (X11/Wayland).
 - OS: Mint 21.2
 - GNOME Shell: 42.9
 - Session type: Wayland

**Steps to Reproduce**
Please provide the steps to reproduce the bug you are encountering.
1. Just start playing wallpaper

**Expected Behavior**
Low cpu usage.

**Actual Behavior**
For some reason it randomly loads random CPU cores to 100%, framerate starts to drop, then loads another core to 100% and all over again.

**Additional Information**
I have clapper installed. not through flatpak, but through a .deb file
![image](https://github.com/jeffshee/gnome-ext-hanabi/assets/133172701/598de318-50d0-465a-81c3-417b4bd957b0)

![image](https://github.com/jeffshee/gnome-ext-hanabi/assets/133172701/151c5ddb-17e7-43fc-8ec9-69043fb74506)

gst-inspect-1.0 nvcodec reports 16 features

**Comment by Renen12** [comment]
Yeah, the same video in a browser window runs even better than on the desktop.

**Comment by shootie22** [comment]
I'm experiencing the same issue, but on Fedora 39, Gnome 45, Wayland on Nvidia

**Comment by Legincy** [comment]
I have the same issues on Fedora 39, GNOME 45.5, Wayland on Nvidia. Turning on the experimental VA-Plugin results into black screen.

**Comment by AsuToki360** [comment]
Same issue here, but I'm no Nvidia user, weird.

Gnome 48, Wayland with Mesa AMDGPU drivers.
